src/db/tables/subscribers.ts

```diff
@@ -55,6 +55,22 @@ async function getSubscriberByFxaUid(
 }
 /* c8 ignore stop */
 
+// Not covered by tests; mostly side-effects. See test-coverage.md#mock-heavy
+/* c8 ignore start */
+async function getSubscriberByOnerepProfileId(
+  onerep_profile_id: SubscriberRow["onerep_profile_id"],
+): Promise<undefined | (SubscriberRow & WithEmailAddresses)> {
+  const [subscriber] = await knex("subscribers").where({
+    onerep_profile_id,
+  });
+  if (!subscriber) {
+    return;
+  }
+  const subscriberAndEmails = await joinEmailAddressesToSubscriber(subscriber);
+  return subscriberAndEmails;
+}
```

Comment:
I don't think we need the associated email addresses, right? And outside of the admin use case, we probably don't want to use this function.

```suggestion
async function getSubscriberByOnerepProfileId(
  onerep_profile_id: SubscriberRow["onerep_profile_id"],
): Promise<undefined | SubscriberRow> {
  const subscriber = await knex("subscribers").first().where({
    onerep_profile_id,
  });
  return subscriber;
}
```

Reply:
That’s right: [7c425dc](https://github.com/mozilla/blurts-server/pull/5850/commits/7c425dcc9eff9b318062e399944d896e38d948c4).